inc/taskjoblog.class.php

```diff
@@ -899,40 +686,33 @@ function getDivState($state, $type='div') {
 
       switch ($state) {
 
-         case 7:
+         case self::TASK_PREPARED:
             return "<".$type." align='center' width='".$width."'>".
                       __('Prepared', 'fusioninventory')."</".$type.">";
 
-         case 1:
+         case self::TASK_STARTED:
             return "<".$type." align='center' width='".$width."'>".
                        __('Started', 'fusioninventory')."</".$type.">";
 
-         case 2:
+         case self::TASK_OK:
             return "<".$type." style='background-color: rgb(0, 255, 0);-moz-border-radius: 4px;".
                      "-webkit-border-radius: 4px;-o-border-radius: 4px;padding: 2px;' ".
                      "align='center' width='".$width."'>".
-                     "<strong>".__('Ok', 'fusioninventory')."</strong></".$type.">";
+                     "<strong>".__('OK')."</strong></".$type.">";
 
-         case 3:
```

Comment:
I guess this is also something that is no longer used now?

Reply:
yes, this taskjoblog state has never been used...